### PR TITLE
adds user's name to email

### PR DESCRIPTION
**Pull request by maxwang7**
Also changes the location of `support_email` in `config.json` (puts it under server)

### CODE REVIEW

**Review by kashizui — APPROVED** [review]
Most people don't provide their real names, in which case their real_name will show up as `None None` -- maybe want to add a bit of logic to handle this, but not necessary since this email is only sent to us.

**Comment by maxwang7** [comment]
Ok, I'll add that logic and merge

File: codalab/rest/help.py [hunk 1]
```diff
@@ -13,18 +13,20 @@
 @post('/help/', apply=AuthenticatedPlugin(), skip=UserVerifiedPlugin)
 def fetch_help():
     message = request.json['message']
-    if 'support_email' not in local.config:
+
+    if local.config and 'support_email' not in local.config['server']:
```
Comment (kashizui) [inline codalab/rest/help.py]: when will config be falsey?

Comment (kashizui) [reply codalab/rest/help.py]: do you mean `if 'server' in local.config ...`?

Comment (maxwang7) [reply codalab/rest/help.py]: Right... would it make sense to check for local.config['server']?

Comment (maxwang7) [reply codalab/rest/help.py]: Or is that first check just not necessary?

Comment (kashizui) [reply codalab/rest/help.py]: I have a similar check elsewhere, so I think it's good to have for now.

Comment (maxwang7) [reply codalab/rest/help.py]: Ok, sounds good

**Pull request merged**